lib/engine/game/g_1812/step/buy_train.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require_relative '../../../step/base'
+require_relative '../../g_1867/step/buy_train'
+require_relative '../../../step/automatic_loan'
+
+module Engine
+  module Game
+    module G1812
+      module Step
+        class BuyTrain < G1867::Step::BuyTrain
+          include Engine::Step::AutomaticLoan
+
+          def pass!
+            Engine::Step::BuyTrain.instance_method(:pass!).bind_call(self)
```

Comment:
This is a bit clunky. Could you just call `super`, and override `nationalize!` in the game class so that it does nothing?

Reply:
that is much simpler, yes! done.